build.sbt

```diff
@@ -13,5 +13,16 @@ libraryDependencies ++= Seq(
   // more dependencies here
   "io.circe" %% "circe-generic-extras" % "0.12.2",
   "io.circe" %% "circe-parser" % "0.12.2",
-  "com.dripower" %% "play-circe" % "2812.0", // compatible with Play 2.8.x
+  "com.dripower" %% "play-circe" % "2812.0" // compatible with Play 2.8.x
 )
+libraryDependencies ++= Seq(
+  // Start with this one
+  "org.tpolecat" %% "doobie-core" % "1.0.0-RC1",
+  // And add any of these as needed
+  "org.tpolecat" %% "doobie-postgres" % "1.0.0-RC1" // Postgres driver 42.3.1 + type mappings.
```

Comment:
Are these comments left over from the doobie tutorial? If so they are not relevant